include/trajopt/solution/SwerveSolution.h

```diff
@@ -21,3 +25,69 @@ struct TRAJOPT_DLLEXPORT SwerveSolution : HolonomicSolution {
 };
 
 }  // namespace trajopt
+
+/**
+ * Formatter for SwerveSolution.
+ */
+//! @cond Doxygen_Suppress
+template <>
+struct fmt::formatter<trajopt::SwerveSolution> {
```

Comment:
Could this use https://en.cppreference.com/w/cpp/utility/format/formatter instead of fmt?

Reply:
Yeah it will. There's build system/choreo compat issues with C++23 right now, and this is still very much a draft (I primarily opened it now just to check the diff actually).